_includes/scenarios/integrate/process-received-enmeshed-onboarding-package-and-create-relationship.md

```diff
@@ -0,0 +1,402 @@
+In this guide we will explain how two Connectors can establish an active Relationship with each other if one of them, the so-called Templator Connector, has prepared an onboarding package and made it available to the other Connector already. The other Connector, the so-called Requestor Connector, must use the [onboarding package]({% link _docs_integrate/process-received-enmeshed-onboarding-package-and-create-relationship.md %}#received-enmeshed-onboarding-package) to [send a Relationship Request]({% link _docs_integrate/process-received-enmeshed-onboarding-package-and-create-relationship.md %}#send-relationship-request) to the Templator Connector. This Relationship Request can then be accepted by the Templator Connector in order to [establish an active Relationship]({% link _docs_integrate/process-received-enmeshed-onboarding-package-and-create-relationship.md %}#establish-active-relationship) between them.
+
+<!--- TODO: Link "Prepare enmeshed onboarding package" einfügen --->
+
+## Received enmeshed onboarding package
+
+As described in detail in the Prepare enmeshed onboarding package guide, our starting situation is that the Templator Connector has created a [RelationshipTemplate]({% link _docs_integrate/data-model-overview.md %}#relationshiptemplate) and the Requestor Connector has successfully loaded it onto itself by proceeding as documented in the [Load RelationshipTemplate created by others]({% link _docs_use-cases/use-case-transport-load-relationship-template-created-by-others.md %}) use case. In particular, the Requestor Connector has get the following result, whereby the notation `<...>` is used as usual as a placeholder for the actual data:
+
+<!--- `POST /api/v2/RelationshipTemplates/Peer` --->
+<!--- TODO: Link "Prepare enmeshed onboarding package" einfügen --->
+
+<!--- ```jsonc
+{
+  "result": {
+    "id": "<ID of RelationshipTemplate>",
+    "isOwn": false,
+    "createdBy": "<Address of Templator Connector>",
+    "createdByDevice": "<ID of Device used for creating RelationshipTemplate>",
+    "createdAt": "<creation date of RelationshipTemplate>",
+    "expiresAt": "<expiration date of RelationshipTemplate>",
+    "content": {
+      //Content of the RelationshipTemplate
+      ...
+    },
+    "truncatedReference": "<truncated reference of RelationshipTemplate>",
+    "maxNumberOfAllocations": <maximum number of allocations>,
+    "secretKey": "<secret key of RelationshipTemplate>"
+  }
+}
+``` --->
```

Comment:
many things commented out and some TODOs. should I wait with the review until this is sorted out?

Reply:
I have removed the comments now (they were only leftovers from the creation process of the documentation), sorry for the confusion.

The only left TODO is now to insert a link to a guide that isn't published yet (but is an open PR (https://github.com/nmshd/documentation/pull/167) and requires a review too) as this documentation depends on other use cases and scenarios.